app/code/Magento/Sales/Model/Order/Address.php

```diff
@@ -508,220 +508,215 @@ public function getVatRequestSuccess()
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setCountryId($id)
     {
         return $this->setData(OrderAddressInterface::COUNTRY_ID, $id);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setFirstname($firstname)
     {
         return $this->setData(OrderAddressInterface::FIRSTNAME, $firstname);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setAddressType($addressType)
     {
         return $this->setData(OrderAddressInterface::ADDRESS_TYPE, $addressType);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setPrefix($prefix)
     {
         return $this->setData(OrderAddressInterface::PREFIX, $prefix);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setMiddlename($middlename)
     {
         return $this->setData(OrderAddressInterface::MIDDLENAME, $middlename);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setSuffix($suffix)
     {
         return $this->setData(OrderAddressInterface::SUFFIX, $suffix);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setCompany($company)
     {
         return $this->setData(OrderAddressInterface::COMPANY, $company);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setVatId($id)
     {
         return $this->setData(OrderAddressInterface::VAT_ID, $id);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setVatIsValid($vatIsValid)
     {
         return $this->setData(OrderAddressInterface::VAT_IS_VALID, $vatIsValid);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setVatRequestId($id)
     {
         return $this->setData(OrderAddressInterface::VAT_REQUEST_ID, $id);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setRegionCode($regionCode)
     {
         return $this->setData(OrderAddressInterface::KEY_REGION_CODE, $regionCode);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setVatRequestDate($vatRequestDate)
     {
         return $this->setData(OrderAddressInterface::VAT_REQUEST_DATE, $vatRequestDate);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setVatRequestSuccess($vatRequestSuccess)
     {
         return $this->setData(OrderAddressInterface::VAT_REQUEST_SUCCESS, $vatRequestSuccess);
     }
 
     /**
-     * {@inheritdoc}
-     *
-     * @return \Magento\Sales\Api\Data\OrderAddressExtensionInterface|null
+     * @inheritdoc
      */
     public function getExtensionAttributes()
     {
         return $this->_getExtensionAttributes();
     }
 
     /**
-     * {@inheritdoc}
-     *
-     * @param \Magento\Sales\Api\Data\OrderAddressExtensionInterface $extensionAttributes
```

Comment:
Please, don't remove type hints for the method because the parent method has annotation with generic interface